common/app/myaccount/CustomerSessionKeys.java

```diff
@@ -1,10 +0,0 @@
-package myaccount;
-
-public final class CustomerSessionKeys {
```

Comment:
It made sense if somebody outside the session utils is actually using them, but nobody is accessing them, so the separation is just creating more classes (3 more). That's why after creating the OrderSessionUtils and realizing the usage outside the class is null, I decided to merge them. Of course once Sunrise is a framework it will be likely used outside, that's why I would rather rename the constants to something that indicates the meaning. Therefore if somebody is not finding the method they are looking for in the *SessionUtils, they can simply access the session key in the same class, instead of guessing they will find them in another class.